### PR TITLE
[4.4] Fix error editor not found in contenthistory modal

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #42445 .

### Summary of Changes
Show the user_id in created_by if the user is not found. 

### Testing Instructions
see #42445

### Actual result BEFORE applying this Pull Request
see #42445

### Expected result AFTER applying this Pull Request

grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_contenthistory/tmpl/history/modal.php [hunk 1]
```diff
@@ -100,7 +100,7 @@
                             <?php endif; ?>
                         </td>
                         <td class="d-none d-md-table-cell">
-                            <?php echo htmlspecialchars($item->editor); ?>
+                            <?php echo empty($item->editor) ? ((int) $item->editor_user_id) : htmlspecialchars($item->editor); ?>
```
Comment (Quy) [inline administrator/components/com_contenthistory/tmpl/history/modal.php]:
```suggestion
                            <?php echo empty($item->editor) ? $item->editor_user_id : htmlspecialchars($item->editor); ?>
```

Comment (Quy) [reply administrator/components/com_contenthistory/tmpl/history/modal.php]: Change `(int) $item->editor_user_id)` to `var_dump($item->editor_user_id)` to see it is already an integer and no need to cast.

42458-integer

Comment (chmst) [reply administrator/components/com_contenthistory/tmpl/history/modal.php]: Strange, I commited the suggestion but this was lost, sorry.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3e51f370587667c756f34a238ee846d1f3a05262<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42458">issues.joomla.org/tracker/joomla-cms/42458</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 3e51f370587667c756f34a238ee846d1f3a05262<br /><br />Thanks @chmst <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42458">issues.joomla.org/tracker/joomla-cms/42458</a>.</sub>

**Pull request merged**